pkg/apis/batch/validation/validation.go

```diff
@@ -66,6 +66,11 @@ const (
 
 	// maximum length of the value of the managedBy field
 	maxManagedByLength = 63
+
+	// maximum length of succeededIndexes in JobSuccessPolicy.
+	maxJobSuccessPolicySucceededIndexesLimit = 64 * 1024
+	// maximum number of rules in successPolicy.
+	maxSuccessPolicyRule = 20
```

Comment:
I would go much lower... maybe 8?

Reply:
I don't have any strong point of view.
Just I adopted the same number of podFailurePolicy.

@alculquicondor If so, do you still prefer to set `8`?

Reply:
oh, we used such a big number there... ok, let's keep them similar.

Reply:
Sure